app/docs/getting-started-with-dockstore.md

```diff
@@ -61,6 +61,8 @@ Tools can be registered manually from the 'My Tools' page by pressing the 'Add T
 
 The Source Code Repository and Image Registry fields must be filled out, they are in the format `namespace/name` (the two paths may differ). The Dockerfile Path,  CWL/WDL Descriptor Paths, and CWL/WDL Test Parameter Paths are relative to the root of the Source Code Repository (and must begin with '/'), these will be the default locations to find their corresponding files, unless specified otherwise in the tags. The toolname is an optional 'suffix' appended to the Dockstore path, it allows for two repositories to share the same Git and Image Registry paths; the combination of Docker image registry path and toolname uniquely distinguishes tools in Dockstore.
 
+If the tool you are registering is private, meaning that to access it on the image registry requires authentication, then click the private checkbox. You will also be asked for a tool maintainer email. This is the email of the person who will be responsible for giving access to the image registry entry to users who request access so that the users can launch the tool. If you do not provide a tool maintainer email, we will use the email found in the CWL descriptor of the tool as the email that a user would contact to request access.
```

Comment:
This seems a bit long and awkward. Maybe reverse it `If you want to register a private Docker image and manage access, please click the "private" checkbox. ...`